pkg/cmd/grafana-cli/commands/reset_password_command.go

```diff
@@ -16,7 +19,20 @@ import (
 const AdminUserId = 1
 
 func resetPasswordCommand(c utils.CommandLine, sqlStore *sqlstore.SqlStore) error {
-	newPassword := c.Args().First()
+	newPassword := ""
+
+	if c.Bool("passwordFromStdin") {
```

Comment:
```suggestion
	if c.Bool("password-from-stdin") {
```

Reply:
Good suggestion! I decided to use the camelCase style for this flag because other flags of grafana-cli are written using camelCase style. Is it okay to use kebab-case there?
https://github.com/grafana/grafana/blob/master/pkg/cmd/grafana-cli/main.go#L44

Reply:
Oh, there is the answer above. Sorry :)